azure-pipelines.yml

```diff
@@ -237,7 +237,7 @@ jobs:
       )
     matrix:
       debian_atlas_32bit:
-        DOCKER_CONTAINER: 'i386/debian:12'
+        DOCKER_CONTAINER: 'i386/debian:sid-20240812'
```

Comment:
I am wondering whether it would not be a tiny bit less risky to use testing rather than unstable, i.e. trixie rather than sid. My understanding following the [doc](https://www.debian.org/doc/manuals/debian-faq/choosing.en.html#s3.1) is that testing is a bit less likely to break than unstable.

Reply:
Yes but since I used a fixed date-based snapshot of SID I think it's fine, no? The tests seems to be green.

Reply:
OK I am not 100% sure, but since we do `apt-get update` and `apt-get install <plenty-of-stuff>` I would expect that we get whatever latest packages are available in `sid`.

Reply:
You are right. Let me try with trixie image then.